app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -33,6 +33,10 @@ def provider_object(connection = nil)
     (connection || connection_source.connect).api.credentials.find(manager_ref)
   end
 
+  def native_ref
```

Comment:
Any reason to introduce this rather than override the `manager_ref` method?

Reply:
That was my initial apporach. Pinged you from an outdated comment thread about that approach.